demos/python_demos/clip_image_classification/download_model_requirements.txt

```diff
@@ -1,5 +1,5 @@
 --extra-index-url https://download.pytorch.org/whl/cpu
-openvino>=2023.2.0
+openvino==2023.3.0
```

Comment:
Why is this needed? Now we will need to manually update requirements with every release.

Reply:
Okay, I see it's a PR to the release branch